### PR TITLE
DRILL-7155: Create a standard logging message for batch sizes generated by indivi…

### PR DESCRIPTION
**Pull request by rhou1**
…dual operators.

This is needed for QA verification of the Batch Size feature DRILL-6238.

@bitblender @sachouche can you please review this PR?

### CODE REVIEW

File: exec/java-exec/src/main/java/org/apache/drill/exec/util/record/RecordBatchStats.java [hunk 1]
```diff
@@ -271,6 +271,19 @@ public static String printAllocatorStats(BufferAllocator allocator) {
...
 
+  /**
+   * Prints the configured batch size
+   *
+   * @param batchStatsContext batch stats context object
+   * @param batchSize contains the configured batch size
+   */
+  public static void printConfiguredBatchSize(RecordBatchStatsContext batchStatsContext,
+    int batchSize) {
+
```
Comment (bitblender) [inline exec/java-exec/src/main/java/org/apache/drill/exec/util/record/RecordBatchStats.java]: _logRecordBatchStats(...)_ does nothing if _batchStatsContext.isEnableBatchSzLogging()_ is false. The overhead of creating _message_ and the call overhead of _logRecordBatchStats(...)_ is wasted if that is the case. Please add a check and avoid the overhead.

**Comment by arina-ielchiieva** [comment]
@rhou1 please create Jira for this fix and add Jira number to the PR title and commit message.

**Comment by rhou1** [comment]
I have updated this PR with the requested changes.  @bitblender @arina-ielchiieva can you please review the changes?

File: exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/columnreaders/batchsizing/RecordBatchSizerManager.java [hunk 2]
```diff
@@ -336,8 +336,7 @@ private long normalizeMemorySizePerBatch() {
     }
 
     if (batchStatsContext.isEnableBatchSzLogging()) {
-      final String message = String.format("The Parquet reader batch memory has been set to [%d] byte(s)", normalizedMemorySize);
-      RecordBatchStats.logRecordBatchStats(message, batchStatsContext);
+      RecordBatchStats.printConfiguredBatchSize(batchStatsContext, (int)normalizedMemorySize);
```
Comment (arina-ielchiieva) [inline exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/columnreaders/batchsizing/RecordBatchSizerManager.java]:
```suggestion
      RecordBatchStats.printConfiguredBatchSize(batchStatsContext, (int) normalizedMemorySize);
```

Comment (rhou1) [reply exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/columnreaders/batchsizing/RecordBatchSizerManager.java]: Done

**Comment by rhou1** [comment]
@arina-ielchiieva I updated the code.  Can you please review the change.

**Comment by arina-ielchiieva** [comment]
+1

**Pull request closed without merging**